deploy/charts/library/templates/_cluster-rolebinding.tpl

```diff
@@ -67,7 +67,7 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-mgr-system{{ template "library.suffix-cluster-namespace" . }}
-  namespace: {{ .Values.operatorNamespace | default .Release.Namespace }} # namespace:operator
+  namespace: {{ .Values.operatorNamespace | default .Release.Namespace }} # namespace:cluster
```

Comment:
hmm, if the helm chart is using the operator namespace, perhaps this was correct. I'll take a deeper look later...

Reply:
Ok, the conclusion is that this RoleBinding namespace should remain as the operator namespace. This gives the mgr access to the configmaps in the operator's namespace, even while the mgr is running in another namespace. So the RoleBinding is needed in the operator namespace, and the `subjects` contains the service account of the mgr in the other namespace. 

Also notice that the `name` of this RoleBinding in the helm chart includes a suffix depending on the cluster namespace. This is to avoid a role binding with the same name that would cause a collision as we saw in the CI of the other change. So let's revert these changes and then make the following changes to the [second cluster rbac](https://github.com/rook/rook/blob/master/deploy/examples/common-second-cluster.yaml#L54-L58):
```yaml
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: rook-ceph-mgr-system-secondary
  namespace: rook-ceph # namespace:operator
```

The differences are that there is a suffix on the `name` to avoid the conflict with the binding created for the first cluster, and then change the namespace back to the operator's namespace.

Reply:
@travisn thank you for investigating this, I'll close this PR and adjust the other one (to fix the `common-second-cluster.yaml` file)